pkg/model/config/config.go

```diff
@@ -118,13 +118,15 @@ type GVK struct {
 	Group   string `json:"group,omitempty"`
 	Version string `json:"version,omitempty"`
 	Kind    string `json:"kind,omitempty"`
+	Plural  string `json:"plural,omitempty"`
```

Comment:
we need a test to ensure these changes. WDYT about add/update the testdata samples with this scenario?

Reply:
No idea about the addon scenario, but I guess it shouldn't affect as it was being passed a `Resource` which already ahs this field (it was always set with `flect.Pluralize(strings.ToLower(resource.Kind))` instead of providing the user the ability to define it).

Will add the test later.